bpf/cpu/cpu.bpf.c

```diff
@@ -870,6 +951,8 @@ int walk_user_stacktrace_impl(struct bpf_perf_event_data *ctx) {
     // > stack frame by setting the frame > pointer to zero.
     //
     // https://refspecs.linuxbase.org/elf/x86_64-abi-0.99.pdf
+    LOG("[debug] reachy reachy bottom %d", unwind_state->jitted);
```

Comment:
stray comment? 😄 It will always be false here anyways, no? We always expect to go from the JIT -> AoT transition